queries/water-z11.pgsql

```diff
@@ -35,6 +35,24 @@ FROM
         AND way_area::bigint > 25600 -- 4px
         AND mz_way12 && !bbox!
 
+    --
+    -- Water line geometries
+    --
+    UNION
+
+    SELECT name,
+           NULL AS area,
```

Comment:
We have access to the area here. Might as well use it for all the queries.

    way_area::bigint AS area